apps/betterangels-backend/shelters/tests/test_models.py

```diff
@@ -0,0 +1,72 @@
+from os import name
+
+from django.test import TestCase
+from pghistory.models import Events
+from shelters.enums import GeneralServiceChoices, PopulationChoices, ShelterChoices
+from shelters.models import GeneralService, Population, Shelter, ShelterType
+
+
+class ShelterModelTestCase(TestCase):
+
+    def setUp(self) -> None:
+        self.shelter = self._create_shelter("Test Shelter")
```

Comment:
tiny nitpick: since the `_create_shelter` private method is only used once, can just pop the entire function body into this `setUp` function